app/src/main/res/layout-land/activity_settings.xml

```diff
@@ -7,7 +7,7 @@
     android:orientation="vertical"
     android:theme="@style/SettingsTheme">
 
-    <fragment
+    <androidx.fragment.app.FragmentContainerView
```

Comment:
how come we're using FragmentContainerView instead of fragment? just curious :]

Reply:
This was related to the comments Matt Corbett made in #29 - `FragmentContainerView` is the new preferred method of adding fragments to an activity.

Reply:
neat!